### DISCUSSION TITLE
Having issues with jest/test/.spec. files and missing nuxtignore

### DISCUSSION THREAD

**Discussion by SKlarsen84** [nuxt/nuxt#17890] [Questions]
Hi we're having a lot of trouble getting nuxt3 to play nice with our jest test files. Essentially we are having trouble getting the build process to NOT include *.test.* and *.spec.* files to the build, which ultimately breaks our app.

In nuxt2 the solution would be to simply add these file types to the .nuxtignore but as I can gather, no such option exists currently for nuxt3. Does anybody have any experience with this problem, and  any ideas how to solve it that doesn't require us to actually delete the tests and spec files from our app?

**Comment by danielroe** [comment]
This should now be possible with https://github.com/nuxt/framework/pull/3424 - spec/test files are excluded by default, and `.nuxtignore` can be configured as in nuxt 2.
